examples/demo/src/app.rs

```diff
@@ -46,6 +47,7 @@ impl Hooks for App {
             Box::new(initializers::axum_session::AxumSessionInitializer),
             Box::new(initializers::view_engine::ViewEngineInitializer),
             Box::new(initializers::hello_view_engine::HelloViewEngineInitializer),
+            Box::new(loco_extras::initializers::normalize_path::NormalizePathInitializer),
```

Comment:
Can you please add a test case in `https://github.com/loco-rs/loco/tree/master/examples/demo/tests/requests`?

Reply:
Added tests using the `ping` endpoint.